modules/fgbio/fastqtobam/main.nf

```diff
@@ -9,29 +9,30 @@ process FGBIO_FASTQTOBAM {
 
     input:
     tuple val(meta), path(reads)
-    val read_structure
 
     output:
-    tuple val(meta), path("*_umi_converted.bam"), emit: umibam
-    path "versions.yml"                         , emit: versions
+    tuple val(meta), path("*.bam"), emit: bam
+    path "versions.yml"           , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def sample_name = args.contains("--sample") ? "" : "--sample ${prefix}"
+    def library_name = args.contains("--library") ? "" : "--library ${prefix}"
```

Comment:
hm intuitively I would say they should go to args, but I  have been seeing some modules that have been more lenient on this :D

Reply:
Both those options are mandatory, so that's why they're defined in the module, with a fallback to `ext.args`